scripts/overrides/index.js

```diff
@@ -2,7 +2,7 @@ import { readFileSync } from "fs";
 import { resolve, join, dirname } from "path";
 import { fileURLToPath } from "url";
 
-const SUPPORTED_GHES_OPERATIONS = ["3.9", "3.10", "3.11", "3.12"];
+const SUPPORTED_GHES_OPERATIONS = ["3.10", "3.11", "3.12", "3.13","3.14"];
```

Comment:
For what it's worth, 3.10 is now [officially deprecated as well](https://docs.github.com/en/enterprise-server@3.12/admin/all-releases).

Reply:
:sweat: It was still supported 3 days ago lol